openai/wandb_logger.py

```diff
@@ -292,9 +306,9 @@ def _log_artifact_inputs(cls, file, prefix, artifact_type, project, entity):
             # log number of items
             wandb.config.update({f"n_{prefix}": artifact.metadata.get("items")})
 
-        wandb.run.use_artifact(artifact, aliases=["latest", artifact_alias])
+        wandb.run.use_artifact(artifact, aliases=["latest", artifact_alias])  # type: ignore
```

Comment:
I'm unsure about `type: ignore`, but it's the best concise solution I've come up with.